.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,23 @@
+name: npm-publish
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  npm-publish:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@master
```

Comment:
It seems like the standard is to use major tag versions like `actions/checkout@v2` to protect against backwards incompatible changes breaking your action. If an action is using major tag versions, I'd use it, as upgrading is supposed to be intentional and test between major versions.